indico/modules/events/abstracts/operations.py

```diff
@@ -211,7 +211,7 @@ def judge_abstract(abstract, abstract_data, judgment, judge, contrib_session=Non
     if judgment == AbstractAction.accept:
         abstract.state = AbstractState.accepted
         abstract.accepted_track = abstract_data.get('accepted_track')
-        if abstract_data.get('override_contrib_type'):
+        if abstract_data.get('override_contrib_type') or abstract_data.get('accepted_contrib_type'):
```

Comment:
Its a boolean that we pass when we select override type in bulk judgment 
![overr](https://cloud.githubusercontent.com/assets/9071624/25428323/cf2fdc0a-2a75-11e7-8b39-a4afc3563e3b.png)
But when doing the single judgment we dont have this setting therefore it was always fallsy even when you selected a type to accept
![ov](https://cloud.githubusercontent.com/assets/9071624/25428422/18311982-2a76-11e7-869e-035c5c6d9370.png)